vizro-core/docs/pages/user-guides/customizing-css.md

```diff
@@ -0,0 +1,330 @@
+# Customizing CSS
+
+Vizro is quite opinionated when it comes to visual formatting, but it allows for some customization if you want to
+deviate from the default settings. While certain elements, such as the layout of the navigation and components,
+are fixed, you can customize background colors, fonts, and other styles via CSS overrides.
+
+To do so, you need to:
+
+1. Add a CSS file to your `assets` folder, refer to our user guide on [adding static assets](assets.md#how-to-add-static-assets).
+2. Identify the right CSS selector for the component you want to style.
+3. Add the desired changed properties to your CSS file.
+
+
+
+## Intro to CSS
+Customizations are typically made using CSS. For a quick 5-minute introduction to CSS, we recommend reading
+this article: [Get Started with CSS in 5 Minutes.](https://www.freecodecamp.org/news/get-started-with-css-in-5-minutes-e0804813fc3e/)
+
+For a more comprehensive tutorial, refer to the [W3Schools CSS tutorial](https://www.w3schools.com/css/default.asp).
+The entire tutorial is beneficial, but the section on [CSS selectors](https://www.w3schools.com/css/css_selectors.asp)
+will be particularly useful.
+
+In Vizro, the CSS file is read in as an external stylesheet. The most common way of applying any styling to Vizro is therefore
+through the use of CSS selectors:
+
+- **Element Selector**: Applies the style to all elements inside the Vizro app.
+
+    ```
+    h1 {
+        font-size: 20px;
+    }
+
+    p {
+        color: green;
+    }
+    ```
+
+- **Class selector:** Applies the style to all elements with the specified class.
+
+    ```
+    .card {
+        background: lightblue;
+    }
+    ```
+
+- **ID selector:** Applies the style to the element with the specified ID.
+
+    ```
+    #my-card {
+        background: lightblue;
+    }
+    ```
+
+
+## Identify the right CSS selector
+
+Use Chrome DevTools or a similar tool (Web Inspector, Web Developer Tools, etc.) to inspect the HTML document in your browser.
+
+1. **Open DevTools:** In Google Chrome, right-click on the app and select "Inspect" from the context menu. This will open the
```

Comment:
```suggestion
1. **Open DevTools:** In Google Chrome, right-click on the app and select "Inspect" from the context menu. This opens the
```

Reply:
Ahh, that's a good advice. Let me change it throughout all the docs I see!